.prettierignore

```diff
@@ -3,5 +3,8 @@
 out/
 pnpm-lock.yaml
 dist/
-website/remote-files/*.json
 website/.graphclient/
+
+# ignore below md since they are fetched from GitHub
+website/pages/en/developing/graph-ts/*.md
+website/pages/en/querying/graph-client/*.md
```

Comment:
Same comment as above about `.mdx`

Reply:
since they are fetched from GitHub we don't need to format them or fix lint errors